airbyte-config/config-models/src/main/resources/types/ConnectorBuilderProject.yaml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/ConnectorBuilderProject.yaml
+title: ConnectorBuilderProject
+description: low-code cd project editable in the connector builder UI
+type: object
+required:
+  - name
+  - builderProjectId
```

Comment:
i don't know if this is a requirement, but the row's identifier is usually just `id`

Reply:
it is in the table, but it doesn't have to be in the config object passed around in the application layer (e.g. https://github.com/airbytehq/airbyte/blob/b9e3c8a7f26b5fb108cc63a6f2a9878ad062547d/airbyte-config/config-models/src/main/resources/types/StandardSourceDefinition.yaml#L16 even though the column in the database is `id`)